python/remage/cli.py

```diff
@@ -264,26 +361,43 @@ def remage_run_from_args(
     # output post-processing (merging multiple LH5 files)
     output_files = ipc_info.get("output")
     main_output_file = ipc_info.get_single("output_main", None)
-    overwrite_output = ipc_info.get_single("overwrite_output", "0") == "1"  # noqa: F841
+    overwrite_output = ipc_info.get_single("overwrite_output", "0") == "1"
     registered_detectors = ipc_info.get("detector", 3)  # noqa: F841
+
     # we might have no output file.
     if len(output_files) > 1 and main_output_file is not None:
         assert main_output_file not in output_files
         output_file_exts = {
             Path(p).suffix.lower() for p in [*output_files, main_output_file]
         }
         if output_file_exts == {".lh5"}:
-            pass
-            # lh5.concat.lh5concat(
-            #     lh5_files=output_files,
-            #     output=main_output_file,
-            #     overwrite=overwrite_output,
-            # )
-            # set the merged output file for downstream consumers.
-            # ipc_info.set("output", [main_output_file])
-            # delete un-merged output files.
-            # for f in output_files:
-            #     Path(f).unlink()
+            # merge output files if requested
+            if py_args.merge_output_files and py_args.reshape_output:
+                # config get_rebooost_config(reshape_table_list,other_table_list,time_window = args.time_window_in_us)
+                # build_hit(
+                #    config
+                #    {},
+                #    stp_files = output_files,
+                #    glm_files = None,
+                #    hit_files = main_output_file
+                # )
+                msg = "Reshaping output files is not implemented yet"
+                raise NotImplementedError(msg)
+
+            lh5.concat.lh5concat(
```

Comment:
the logic of this block seems wrong, please review. also info message about merging in progress?